docs/modules/user-guide/partials/proc_updating-the-devfile-library-schema.adoc

```diff
@@ -0,0 +1,13 @@
+[id="proc_updating-the-devfile-library-schema_{context}"]
+= Updating the devfile library schema
+
+Update the schema version to ensure that you develop with the latest code from devfile properties:
```

Comment:
```suggestion
To ensure that you develop with the latest code from devfile properties, update the devfile library schema to the latest version.
```

Reply:
I don't think we have a guideline for this, but I find that putting the motive first and the action last scans better.